### ISSUE TITLE
[CLOSED] Socket Bonus bug

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#1970]
<a href="https://github.com/vandershep"><img src="https://avatars1.githubusercontent.com/u/15201155?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [vandershep](https://github.com/vandershep)**    
_Monday Oct 19, 2015 at 18:36 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/1970_

----

Currently, if you equip an item with gems in order to achieve a socket bonus, and then switch the gems out for better ones (which would normally result in losing the socket bonus), the socket bonus still remains on your stat sheets (and persists through logging out and logging back in). I am not certain whether or not the persisting values are actually being applied to your character, only that they remain on the stat sheet. Below are screenshots providing an example of this bug:

Gemming for socket bonus resulting in 3 hit rating

http://i.imgur.com/UtIDut5.png

Swapping the orange and blue gems for reds, losing the socket bonus, but still retaining the 3 hit rating

http://i.imgur.com/yO0rFgB.png

**Comment by netherspite** [comment]
<a href="https://github.com/Jumpka"><img src="https://avatars2.githubusercontent.com/u/14101592?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Jumpka](https://github.com/Jumpka)**  
_Monday Oct 19, 2015 at 19:05 GMT_

----

Noone reported this so far? I guess it's combination of these reports:
https://github.com/ccshiro/cc-buglist/issues/1659
https://github.com/ccshiro/cc-buglist/issues/1921
https://github.com/ccshiro/cc-buglist/issues/1190

TLDR: You have to socket the right colors when wearing that item, then the socket bonus will stay forever, if you socket it while in your bags, it won't work.

**Comment by netherspite** [comment]
<a href="https://github.com/Doppelmoral"><img src="https://avatars0.githubusercontent.com/u/2018747?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Doppelmoral](https://github.com/Doppelmoral)**  
_Tuesday Oct 20, 2015 at 16:07 GMT_

----

Have you observed the behaviour of this bug when
1) repeating the procedure (will it keep adding ?)
2) zone in and out of a portal
3) log out and back in ?

Valuable information that may help narrowing down the cause.

**Comment by netherspite** [comment]
<a href="https://github.com/Valmere"><img src="https://avatars2.githubusercontent.com/u/2462756?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Valmere](https://github.com/Valmere)**  
_Tuesday Oct 20, 2015 at 21:08 GMT_

----

1) No, stat bonus will not keep stacking if you keep swapping gems. Once it applies the socket bonus to the item it remains. Taking the item on/off doesn't do anything either.

2) Neither portals nor relogging affect the item. The only way to remove the socket bonus after you get it is by taking the item off and then changing the gem. Bonus will only persist if you swap gems with the item equipped.
